### PR TITLE
🔥 backbone elder

### PR DESCRIPTION
**Pull request by rochoa**
Remove core-decorators, aka elder for backbone's model/view/collection.

Now the alternative to elder is `Model.__super__.{method}.call(this, args)`.

### CODE REVIEW

File: src/geo/ui/search.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ var Search = View.extend({
   clean: function() {
     this._unbindEvents();
     this._destroySearchPin();
-    this.elder('clean');
+    Search.__super__.clean.call(this);
```
Comment (alonsogarciapablo) [inline src/geo/ui/search.js]: What about `View.prototype.clean.call(this)`? We're using this pattern already and `__super__` feels a bit hacky to me.

**Comment by alonsogarciapablo** [comment]
I left a comment with an alternative to `__super__`. Other than that... :fire:

File: src/geo/ui/search.js (hunk 1 above)
Comment (viddo) [inline src/geo/ui/search.js]: Official docs recommends using this :point_up: approach too, see end of http://documentcloud.github.io/backbone/#Model-extend probably better than relying on internals that might change in the future.

Comment (rochoa) [inline src/geo/ui/search.js]: :+1: I should have looked into this. Two more reasons to not rely on `__super__`:

 - https://github.com/jashkenas/backbone/pull/3539#issuecomment-82384843
 - https://github.com/jashkenas/backbone/pull/787#issuecomment-3143358

I will take of it.

**Comment by rochoa** [comment]
Are you both OK with latest changes?

**Comment by viddo** [comment]
Sí señor :dancer:

**Comment by alonsogarciapablo** [comment]
:+1:

**Pull request merged**